<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Notice</class>
 <widget class="QMainWindow" name="Notice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:reflect, x1:0.837, y1:0.528682, x2:0.567, y2:0.465682, stop:0.0995025 rgba(193, 224, 239, 255), stop:0.965174 rgba(230, 240, 240, 255));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QPushButton" name="backbt">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>450</y>
      <width>28</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/icon/back.png);
background-color:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="formWidget" native="true">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>80</y>
      <width>281</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(239, 255, 255,220);
border-radius:5px;
font:14px;
border-top: 2px solid QLinearGradient(y0:0, y1:1,stop: 0 #DADADA, stop: 1 transparent);
border-left:2px solid QLinearGradient(x0:0, x1:1,stop: 0 #DADADA, stop: 1 transparent);
border-bottom:2px solid QLinearGradient(y0:0, y1:1,stop: 0 transparent, stop: 1  #DADADA);
border-right: 2px solid QLinearGradient(x0:0, x1:1,stop:  0 transparent, stop: 1 #DADADA);
</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius:0px;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.961, y2:0.971591, stop:0 rgba(0, 0, 0, 0), stop:0.920398 rgba(0, 158, 255, 0));

color:rgb(98, 98, 98);

border:none;</string>
         </property>
         <property name="text">
          <string>标题</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="titleedit">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>18</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>18</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:rgb(98, 98, 98);
border-radius:5px;
background-color:rgb(233, 233, 234,200);</string>
         </property>
         <property name="placeholderText">
          <string>输入标题</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius:0px;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.961, y2:0.971591, stop:0 rgba(0, 0, 0, 0), stop:0.920398 rgba(0, 158, 255, 0));
color:rgb(98, 98, 98);

border:none;</string>
         </property>
         <property name="text">
          <string>发布人</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="notice_name">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>18</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>18</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:rgb(98, 98, 98);
border-radius:5px;
background-color:rgb(233, 233, 234,200);

</string>
         </property>
         <property name="text">
          <string>smx</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius:0px;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.961, y2:0.971591, stop:0 rgba(0, 0, 0, 0), stop:0.920398 rgba(0, 158, 255, 0));

color:rgb(98, 98, 98);

border:none;</string>
         </property>
         <property name="text">
          <string>活动开始</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="start_time">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>18</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>18</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:rgb(98, 98, 98);
border-radius:5px;
background-color:rgb(233, 233, 234,200);</string>
         </property>
         <property name="placeholderText">
          <string>活动开始时间(yyyy-MM-dd)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius:0px;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.961, y2:0.971591, stop:0 rgba(0, 0, 0, 0), stop:0.920398 rgba(0, 158, 255, 0));

color:rgb(98, 98, 98);

border:none;</string>
         </property>
         <property name="text">
          <string>活动结束</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="end_time">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>18</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>18</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:rgb(98, 98, 98);
border-radius:5px;
background-color:rgb(233, 233, 234,200);</string>
         </property>
         <property name="placeholderText">
          <string>活动结束时间(yyyy-MM-dd)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="messageedit">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>230</y>
      <width>281</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:reflect, x1:0.837, y1:0.528682, x2:0.567, y2:0.465682, stop:0.0995025 rgba(217, 233, 246), stop:0.965174 rgba(228, 240, 248));
color: #5b5b5b;
border-top: 2px solid QLinearGradient(y0:0, y1:1,stop: 0 #DADADA, stop: 1 transparent);
border-left:2px solid QLinearGradient(x0:0, x1:1,stop: 0 #DADADA, stop: 1 transparent);
border-bottom:2px solid QLinearGradient(y0:0, y1:1,stop: 0 transparent, stop: 1  #DADADA);
border-right: 2px solid QLinearGradient(x0:0, x1:1,stop:  0 transparent, stop: 1 #DADADA);</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;11&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendbt">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>330</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-image: url(:/icon/send0.png);
background-color: rgb(149, 245, 255,0);
border-radius:0px;
}
QPushButton:pressed{  
border-image: url(:/icon/send1.png);
border-radius:0px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="deletebt">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>410</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-image: url(:/icon/delete0.png);
background-color:rgba(255, 255, 255, 0);
}
QPushButton:pressed{  
border-image: url(:/icon/delete1.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="titleclearedit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>201</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(228, 240, 248);
color:rgb(98, 98, 98);
border-radius:5px;
font:17px;
border-top: 2px solid QLinearGradient(y0:0, y1:1,stop: 0 #e1e1e1, stop: 1 transparent);
border-left:2px solid QLinearGradient(x0:0, x1:1,stop: 0 #e1e1e1, stop: 1 transparent);
border-bottom:2px solid QLinearGradient(y0:0, y1:1,stop: 0 transparent, stop: 1 #e1e1e1);
border-right:2px solid QLinearGradient(x0:0, x1:1,stop:  0 transparent, stop: 1 #e1e1e1);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>通过标题进行删除</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>15</y>
      <width>681</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:reflect, x1:0.837, y1:0.528682, x2:0.567, y2:0.465682, stop:0.0995025 rgba(217, 233, 246), stop:0.965174 rgba(228, 240, 248));
border-radius:10px;
color:rgb(139, 139, 139);
font-weight:150;
font: 19pt;
background-image:url()0;</string>
    </property>
    <property name="text">
     <string>通知栏信息界面</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableView1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>301</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget{
color:rgb(98, 98, 98);
background:rgb(215, 215, 215,100);
alternate-background-color: rgb(180, 180, 180); /*相邻行颜色不同*/
gridline-color:rgb(255, 255, 255,200);
border-top: 2px solid QLinearGradient(y0:0, y1:1,stop: 0 #DADADA, stop: 1 transparent);
border-left:2px solid QLinearGradient(x0:0, x1:1,stop: 0 #DADADA, stop: 1 transparent);
border-bottom:2px solid QLinearGradient(y0:0, y1:1,stop: 0 transparent, stop: 1  #DADADA);
border-right: 2px solid QLinearGradient(x0:0, x1:1,stop:  0 transparent, stop: 1 #DADADA);
}
QTableWidget::item:selected{
background:rgb(180, 180, 180);
}
QTableWidget::item:hover{
background:rgb(180, 180, 180);
}
QTableWidget QTableCornerButton::section {
background: rgb(180, 180, 180,100);
}

QHeaderView::section{
text-align:center;
background-color:rgb(236, 244, 250,200);
padding:2px;
margin:0px;
color: rgb(98, 98, 98,200);
border:1px solid rgb(180, 180, 180,200);
border-left-width:0;

}
/*滑动条*/
QScrollBar:vertical{
background:rgb(180, 180, 180,200);
padding:0px;
border-radius:6px;
max-width:5px;
}
QScrollBar:horizontal {
background:rgb(180, 180, 180,200);
border: 0px;
height: 5px;
}
/*滑块*/
QScrollBar::handle:vertical{
background:rgb(180, 180, 180,150);
}
QScrollBar::handle:horizontal {
background:rgb(180, 180, 180,150);
 }
/*滑块按下*/
QScrollBar::handle:hover:vertical,QScrollBar::handle:pressed:vertical{
background:rgb(170, 170, 170);
}
QScrollBar::handle:hover:horizontal,QScrollBar::handle:pressed:horizontal{
background:rgb(170, 170, 170);
}

/*滑块上下区域样式*/
QScrollBar::sub-page:vertical{
background:rgb(232, 232, 232);
}
QScrollBar::add-page:vertical{
background:rgb(232, 232, 232);
}
QScrollBar::sub-page:horizontal{
background:rgb(232, 232, 232);
}
QScrollBar::add-page:horizontal{
background:rgb(232, 232, 232);
}
/*箭头样式*/
QScrollBar::add-line:vertical{
background:none;
}
QScrollBar::sub-line:vertical{
background:none;
}
QScrollBar::add-line:horizontal {
background:none;
}
QScrollBar::sub-line:horizontal {
background:none;
}
</string>
    </property>
    <property name="columnCount">
     <number>4</number>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>68</number>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>24</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>24</number>
    </attribute>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>4</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>5</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>6</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>7</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>8</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>9</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>10</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>11</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>12</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>活动名称</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>活动内容</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>活动状态</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>发布人</string>
     </property>
    </column>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
